models/zero_point.py

```diff
@@ -41,3 +52,65 @@ class ZeroPoint(Base, AutoIDMixin):
             "and the parameters used to produce this zero point. "
         )
     )
+
+    zp = sa.Column(
+        sa.Float,
+        nullable=False,
+        index=False,
+        doc="Zeropoint: -2.5*log10(adu_psf) + zp = mag"
+    )
+
+    dzp = sa.Column(
+        sa.Float,
+        nullable=False,
+        index=False,
+        doc="Uncertainty on zp"
+    )
+
+    aper_cor_apers = sa.Column(
+        sa.ARRAY( sa.REAL ),
+        nullable=True,
+        default=None,
+        index=False,
+        doc="Pixel radii of apertures whose aperture corrections are in aper_cor_aper."
```

Comment:
How about naming this `aper_cor_radii`?

Reply:
Also I think you meant to say the corrections are in `aper_cor` not `aper_cor_aper`.